keps/sig-api-machinery/4355-coordinated-leader-election/README.md

```diff
@@ -841,8 +843,15 @@ in back-to-back releases.
 -->
 
 #### Alpha
+
 - Feature implemented behind a feature flag
-- The strategy `MinimumCompatibilityVersionStrategy` is implemented
+- The strategy `OldestEmulationVersion` is implemented
+
+#### Beta
```

Comment:
maybe for GA: can we load test this feature? Imagine we create 10000 leases and candidates. What happens?

Reply:
Added a section for GA